include/crm/common/clone_internal.h

```diff
@@ -59,6 +59,20 @@ pcmk__is_unique_clone(const pcmk_resource_t *rsc)
     return pcmk__is_clone(rsc) && pcmk_is_set(rsc->flags, pcmk_rsc_unique);
 }
 
+/*!
+ * \internal
+ * \brief Check whether a resource is an anonymous clone
+ *
+ * \param[in] rsc  Resource to check
+ *
+ * \return true if \p rsc is an anonymous clone, otherwise false
+ */
+static inline bool
+pcmk__is_anonymous_clone(const pcmk_resource_t *rsc)
```

Comment:
Should be a refactor commit, not an API commit

Reply:
Whoops, updated